langchain4j/src/main/java/dev/langchain4j/service/output/ServiceOutputParser.java

```diff
@@ -105,7 +106,7 @@ public String outputFormatInstructions(Type returnType) {
         return "\nYou must answer strictly in the following JSON format: " + jsonStructure((rawClass), new HashSet<>());
     }
 
-    public static String jsonStructure(Class<?> structured, Set<Class<?>> visited) {
+    private static String jsonStructure(Class<?> structured, Set<Class<?>> visited) {
```

Comment:
Minor fix, this method does not need to be public and reducing visibility keeps the public API clean